content/TrueCommand/Administration/Settings/SAML.md

```diff
@@ -42,149 +40,148 @@ To configure AD as the IdP, you must create and modify a Relying Party Trust. Af
 
 * Configure the Claim Issuance Policy to add the incoming and outgoing claim types. 
 
-Each of these top-level steps are described in detail in the example provided below.
+The example below describes each top-level step in detail.
 
 ## Setting Up TrueCommand and AD SAML Service
 
-This procedure assumes the Windows administrator user is QE. Substitute your system addresses (URLs,IP address, port number, names, etc.) where variables are present.
+This procedure assumes the Windows administrator user is QE. Substitute your system addresses (URLs, IP address, port number, names, etc.) where variables are present.
 
-1. Access theTrue Command web interface via http://IP:PORT where IP:PORT is the IP address and port number assigned to your TrueCommand system. 
+1. Access the TrueCommand web interface via *http://IP:PORT* where *IP:PORT* is the IP address and port number assigned to your TrueCommand system. 
 
 2. Go to Config > Administration and select the Configuration tab.
 
-   a. Enter http://ds.yourcompany.net/FederationMetadata/2007-06/FederationMetadata.xml in the SAML Identity Provider URL field, then click Save. The URL is from Active Directory.
+   a. Enter *http://ds.yourcompany.net/FederationMetadata/2007-06/FederationMetadata.xml* in the SAML Identity Provider URL field, then click Save. The URL is from Active Directory.
```

Comment:
The variable is only ds.yourcompany.net the rest is fixed so the italic formatting only applies to the company part of the URL.  and the other on-screen elements should be in bold. Suggest changing to
   a. Enter http://*ds.yourcompany.net*/FederationMetadata/2007-06/FederationMetadata.xml in the **SAML Identity Provider URL** field, then click **Save**. The URL is from Active Directory.

Reply:
Fixed